### DISCUSSION TITLE
Polls in posts

### DISCUSSION THREAD

**Discussion by frank-connolly** [dailydotdev/daily#1234] [Feature Requests]
Recently I had to link to an external site to have a poll in a [squad post](https://app.daily.dev/posts/poll-what-version-of-java-are-you-using--mh7xzh9gn). I would have preferred some daily.dev mechanism for the poll. I think it would improve engagement.

**Comment by idoshamun** [comment]
Hi @frank-connolly,

Thanks for the idea, it was suggested in the past and we constantly talking about it. Unfortunately, it is not prioritized at the moment.
I'm closing this to keep the discussion here: https://github.com/dailydotdev/daily/discussions/1104
